app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] in these lines the foreign key for posts, comments and likes should be set as well.